pkgconfig/arm-vita-eabi-pkg-config

```diff
@@ -11,4 +11,4 @@ export PKG_CONFIG_SYSROOT_DIR=
 export PKG_CONFIG_LIBDIR=${VITASDK}/arm-vita-eabi/lib/pkgconfig
 
 [[ "$1" == '--version' ]] && exec pkg-config --version
-exec pkg-config --static "$@"
+exec pkg-config --define-prefix --static "$@"
```

Comment:
This might work for all the current packages.

However, some packages do install `.pc` files in subdirectories of `pkgconfig/`, I'm not sure if it'll work in that case.

Reply:
Do you have an example of such package? Because if so, then we have another bug: pkg-config won't find them in subdirectory. Although, none of packages on my host system do that, and host pkg-config can't find `.pc` files in subdirectories either.

Reply:
Nevermind, I've just checked my installation and the `.pc` files in subdirectories are *also* symlinked into the `pkgconfig` directory (e.g. for mit-krb5)